src/components/LayerTree/__snapshots__/LayerTree.test.js.snap

```diff
@@ -366,100 +366,21 @@ exports[`LayerTree matches snapshots using default properties. 1`] = `
           <span />
         </label>
         <div
-          aria-expanded={true}
```

Comment:
Why is this snapshot test changing?

Reply:
There was a small modification missing in Treedata. Will fix